k8s/scripts/ceph-csi.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -xe
+
+yum install -y ceph-common
+
+# Deploy RBACs for sidecar containers and node plugins
+kubectl --kubeconfig /etc/kubernetes/admin.conf create -f /tmp/csi-attacher-rbac.yaml
```

Comment:
We should consider adding a ceph subdirectory and do something like `kubectl create -f /tmp/ceph/`. Other components do it right now but it only gets worse if we never start it.